Cargo.lock

```diff
@@ -1,1527 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
The lockfile is needed. It locks all deps to the required/compatible versions.
Is this by accident?